app/views/spree/admin/shipping_methods/new.html.haml

```diff
@@ -0,0 +1,16 @@
+- content_for :page_title do
+  = Spree.t(:new_shipping_method)
+- content_for :page_actions do
+  %li
+    = button_link_to Spree.t(:back_to_shipping_methods_list), spree.admin_shipping_methods_path, :icon => 'icon-arrow-left'
+%div{"data-hook" => "admin_shipping_method_new_form_header"}
+  = render :partial => 'spree/shared/error_messages', :locals => { :target => @shipping_method }
+%div{"data-hook" => "admin_shipping_method_new_form"}
```

Comment:
I think data-hook is only used to make things deface-able. Now that we are moving away from deface, I think we can remove these.
I am not sure about this, it needs to be confirmed by another dev...

Reply:
Yep, looks like it's just for deface: https://github.com/spree/spree/issues/3187

Thanks for the pointer, I've just realised I need to remove those from the Spree I just recreated for the admin orders page. :)